dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java

```diff
@@ -314,7 +318,7 @@ public String getCompoundRawValue() {
         return returnList;
     }
 
-    public List<String> getRawValuesList() {
+    private List<String> getRawValuesList() {
```

Comment:
Yes. I agree that they are inconsistent. I didn't wan't to change `getRawValuesList` because  it returns kind of mixed data between raw and display. And to be honest I'm not sure if it is needed. More explore should be done because it is used in some api endpoints to compare values. Some detail in implementation of this could be used that I am unaware of right now.
We could create a separate task for it.